.github/scripts/python_package_upload/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM docker.io/python:3.9
```

Comment:
what is the purpose of this dockerfile?

Reply:
Updated the PR description

Reply:
that doesn't really answer the question, where is this dockerfile being used? 
are we building this anywhere? maintaining this image anywhere? 

docs on how one should use this? 

right now it's just a random dockerfile sitting in the repo

Reply:
> where is this dockerfile being used? are we building this anywhere? 

Look at `.github/scripts/python_package_upload/package_upload_run.sh` 

> maintaining this image anywhere?

No. Just for testing purposes.

> docs on how one should use this?

By looking at `package_upload_run.sh` do still believe I should document? If yes, I can do it.

> right now it's just a random dockerfile sitting in the repo

Can you elaborate more what exactly it means?

Reply:
I see, I missed that thanks. Do we have to build this every action? My concern is that we have [limited space](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) in the default runners, can we benefit from buildling this and uploading it to a quay repo instead?

Reply:
If we get impacted by the limited space we can find out an alternative. 

By uploading to a quay repo we need to upload a new image version for every KFP release. The current way do it in runtime.

Reply:
works for me!